.travis.yml

```diff
@@ -37,6 +37,7 @@ r_packages:
   - devtools
 
 before_install:
+  - R CMD build .
```

Comment:
I guess this is on purpose, to install it both here and in the `script` session?

Reply:
Yes, practically no time cost...